### PR TITLE
Rename negamax function to alpha_beta

### PR DESCRIPTION
**Pull request by jouae**
Due to changing the original algorithm from negamax to alpha-beta pruning the function name has been modified to "alpha_beta"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,17 @@ To execute a move, enter `[column][row]`. For example:
 
 Press Ctrl-C to exit.
 
+## Quick Start
+
+Make sure your operating system is **Linux** then use the command in shell
```
Comment (jserv) [inline README.md:26]: This program is not Linux specific. It runs on macOS and BSD system. All you need is a full functioned terminal emulator.

Drop this section if you tend to submit a pull request to rename `negamax` function.

**Review by jserv — CHANGES_REQUESTED** [review]
Run `clang-format -i` before submitting.

**Comment by visitorckw** [comment]
Negamax and alpha-beta pruning are not two opposing algorithms. More precisely, the current implementation utilizes negamax with alpha-beta pruning. Therefore, I am wondering if changing the name might actually decrease the program's readability.

**Comment by jouae** [comment]
@ndsl7109256 Yes I can understand your considerations. 
Perhaps I can left a comment in the negamax function in the program to explain that is not origin negamax algorithm?

**Comment by visitorckw** [comment]
I don't have opinions on whether to add comments.
If you believe that comments would be helpful for the code, feel free to do so.

**Pull request closed without merging**